version.rs

```diff
@@ -91,23 +131,25 @@ async fn main() -> std::io::Result<()> {
         });
 
     //let prometheus = PrometheusMetrics::new("api", Some("/metrics"), None);
-    let server = HttpServer::new(move || {
-        App::new()
-            .app_data(Data::new(reader.clone()))
-            .wrap(middleware::Logger::default().exclude("/health"))
-            //.wrap(prometheus.clone())
-            .service(get_versions)
-            .service(get_version)
-            .service(health)
-    })
-    .bind("0.0.0.0:8000")
-    .expect("bind to 0.0.0.0:8000")
-    .shutdown_timeout(5)
-    .run();
+    let app = Router::new()
+        .route("/versions", get(get_versions))
+        .route("/versions/:namespace/:name", get(get_version))
+        .layer(AddExtensionLayer::new(reader.clone()))
+        .layer(TraceLayer::new_for_http())
```

Comment:
Presumably the inclusion of the TraceLayer here is what causes the above routes to get logging here with axum?

Reply:
Correct!